.travis.yml

```diff
@@ -1,5 +1,3 @@
 language: node_js
 node_js:
-  - node
-script:
-  - npm test
\ No newline at end of file
+  - node
```

Comment:
I am sorry, but could you please leave trailing line break so GitHub won't show this icon?